travis-ymls/chectl.travis.yml

```diff
@@ -0,0 +1,28 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : chectl
+# Source Repo         : https://github.com/che-incubator/chectl
+# Travis Job Link     : https://travis-ci.com/github/rajesh-ibm-power/chectl
+# Created travis.yml  : Yes
+# Maintainer          : Rajesh kumar <Rajesh.kumar13@ibm.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ---
+
+
+language: node_js
+node_js:
+   - 10
```

Comment:
Lets test this on latest node versions( https://github.com/ppc64le/build-scripts/pull/508#discussion_r562398080 )